src/docker-it/run_docker_sshd.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# requires command line argument with path to rultor root folder
+if ($1); then
+    echo "Required path to rultor root folder"
```

Comment:
@mkordas This script will be launched by maven plugin and I am not sure what context path will be default. It could be ${project.build.directory} by default. So, I think, input parameter should be here